src/main/java/com/google/cloud/security/scanner/sources/LiveProjectSource.java

```diff
@@ -223,7 +223,10 @@ private boolean refreshProjects(String nextPageToken) throws IOException {
         if (project.getParent() != null) {
           orgId = project.getParent().getId();
         }
-        this.projects.add(new GCPProject(project.getProjectId(), orgId, project.getName()));
+        if (project.getLifecycleState() == null
```

Comment:
nit: purely optional, for multiple conditions, maybe break it up for easier readability, and is flexible in the future to tack on more conditions.

if (project.getLifecycleState() == null) {
  continue;
}
if (project.getLifecycleState().startsWith("DELETE")) {
  continue;
}
this.projects.add(new GCPProject(project.getProjectId(), orgId, project.getName());